.gitlab-ci.yml

```diff
@@ -156,6 +156,8 @@ deploy:sls:branch:
     - ./ci/config-squidex-rules.sh ./backup/rules
     # create-app.py configures sq to new app
     - sq sync in backup
+    # apply schema changes on top of dev backup
+    - sq sync in packages/squidex/schema -t schemas --delete
```

Comment:
hm.... i dont mind adding it to the PR but... do you think maybe this should run on every commit on a non master branch? Imagine a scenario:

1. An engineer raises a branch with some code changes (not schema)
2. That branch is deployed, dev backup restored and no other changes applied (as per the change you are proposing)
3. That engineer than decides to make changes to the schema and pushes a commit
4. No changes are applied as this jobs trigger is a new PR only (right?)

Reply:
This should do just that.
We have this verification step, but every commit inside a PR will pass it.
https://github.com/yldio/asap-hub/pull/876/files/acb2c10995ae13dc5b197168a309436e79997011#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7L14